File: javascript/ehPrimo.js
function ehPrimo(n) {
  if (n < 2) return false;
  for (let i = 2; i <= Math.sqrt(n); i++) {
    if (n % i === 0) return false;
  }
  return true;
}

console.log("--- Número primo ---");
console.log(`17 é primo? ${ehPrimo(17)}`);

File: java/EhPrimo.java
public class EhPrimo {

    public static boolean ehPrimo(int n) {
        if (n < 2) return false;
        for (int i = 2; i <= Math.sqrt(n); i++) {
            if (n % i == 0) return false;
        }
        return true;
    }

    public static void main(String[] args) {
        System.out.println("--- Número primo ---");
        System.out.println("17 é primo? " + ehPrimo(17));
    }
}
